`spdk` now failing to build for me, nixos x86_64:

```
builder for '/nix/store/9xkcqnqg3lkc2499bp2m1gg8a0zx78ya-spdk-21.10.drv' failed with exit code 2; last 10 log lines:
  /nix/store/vzsqcldzq3y8cwj0mr8pbc6x9w1jmfj1-binutils-2.38/bin/ld.bfd: (.text+0x29c8c): undefined reference to `rte_dma_fp_objs'
  /nix/store/vzsqcldzq3y8cwj0mr8pbc6x9w1jmfj1-binutils-2.38/bin/ld.bfd: (.text+0x29ce2): undefined reference to `rte_dma_fp_objs'
  /nix/store/vzsqcldzq3y8cwj0mr8pbc6x9w1jmfj1-binutils-2.38/bin/ld.bfd: /nix/store/qjhb88y7a7cxb7g1fkc868saivrx7pna-dpdk-22.03/lib/librte_vhost.a(vhost_virtio_net.c.o): in function `rte_vhost_poll_enqueue_completed':
  (.text.experimental+0x3db): undefined reference to `rte_dma_fp_objs'
  /nix/store/vzsqcldzq3y8cwj0mr8pbc6x9w1jmfj1-binutils-2.38/bin/ld.bfd: /nix/store/qjhb88y7a7cxb7g1fkc868saivrx7pna-dpdk-22.03/lib/librte_vhost.a(vhost_virtio_net.c.o):(.text.experimental+0x5444): more undefined references to `rte_dma_fp_objs' follow
  collect2: error: ld returned 1 exit status
  make[3]: *** [/build/source/mk/spdk.app.mk:71: /build/source/build/examples/accel_perf] Error 1
  make[2]: *** [/build/source/mk/spdk.subdirs.mk:44: perf] Error 2
  make[1]: *** [/build/source/mk/spdk.subdirs.mk:44: accel] Error 2
  make: *** [/build/source/mk/spdk.subdirs.mk:44: examples] Error 2
```

Might they need to be bumped together?

--- a/pkgs/development/libraries/spdk/default.nix
+++ b/pkgs/development/libraries/spdk/default.nix
@@ -10,9 +10,19 @@
 , libuuid
 , numactl
 , openssl
+, fetchurl
 }:
 
-stdenv.mkDerivation rec {
+let
+  # The old version has some CVEs howver they should not affect SPDK's usage of the framework: https://github.com/NixOS/nixpkgs/pull/171648#issuecomment-1121964568
+  dpdk' = dpdk.overrideAttrs (old: rec {
+    name = "dpdk-21.11";
+    src = fetchurl {
+      url = "https://fast.dpdk.org/rel/${name}.tar.xz";
+      sha256 = "sha256-Mkbj7WjuKzaaXYviwGzxCKZp4Vf01Bxby7sha/Wr06E=";
+    };
+  });
+in stdenv.mkDerivation rec {
   pname = "spdk";
   version = "21.10";
 
@@ -40,7 +50,7 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    cunit dpdk libaio libbsd libuuid numactl openssl ncurses
+    cunit dpdk' libaio libbsd libuuid numactl openssl ncurses
   ];
 
   postPatch = ''
@@ -49,7 +59,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  configureFlags = [ "--with-dpdk=${dpdk}" ];
+  configureFlags = [ "--with-dpdk=${dpdk'}" ];
 
   NIX_CFLAGS_COMPILE = "-mssse3"; # Necessary to compile.
   # otherwise does not find strncpy when compiling

